core/test/unit/middleware/spam-prevention_spec.js

```diff
@@ -19,6 +20,7 @@ describe('Middleware: spamPrevention', function () {
         spyNext = sinon.spy(function (param) {
             error = param;
         });
+        middleware.spamPrevention = rewire('../../../server/middleware/spam-prevention').__get__('spamPrevention');
```

Comment:
by adding  `__get__('spamPrevention')`, you also know facts about internal implementation. (`spamPrevention` is a global variable in the file and you rewire it)

you could delete  `__get__('spamPrevention')` and just rewire the whole file.